### PR TITLE
Add an option to pass client's QPS/burst to e2e framework

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

**Comment by wojtek-t** [comment]
@kubernetes/sig-testing 

LGTM

File: test/e2e/framework.go [hunk 1]
```diff
@@ -74,10 +78,16 @@ type TestDataSummary interface {
 
 // NewFramework makes a new framework and sets up a BeforeEach/AfterEach for
 // you (you can write additional before/after each functions).
-func NewFramework(baseName string) *Framework {
+func NewDefaultFramework(baseName string) *Framework {
+	return NewFramework(baseName, 5, 10)
+}
+
+func NewFramework(baseName string, qps float32, burst int) *Framework {
```
Comment (timothysc) [inline test/e2e/framework.go]: Could we create a params struct and pass that in with defaults?   I'm just thinking that would be cleaner as the configurable params grow.

Comment (wojtek-t) [inline test/e2e/framework.go]: SGTM - removing LGTM.

**Comment by wojtek-t** [comment]
@gmarek - can you please apply @timothysc `s comment?

**Comment by gmarek** [comment]
Done. @wojtek-t PTAL

File: test/e2e/density.go [hunk 2]
```diff
@@ -131,18 +131,15 @@ var _ = Describe("Density", func() {
...
 	BeforeEach(func() {
-		// Explicitly create a client with higher QPS limits.
-		// However, make those at most comparable to components.
-		config, err := loadConfig()
-		Expect(err).NotTo(HaveOccurred())
-		config.QPS = 20
-		config.Burst = 30
-		c, err = loadClientFromConfig(config)
-		Expect(err).NotTo(HaveOccurred())
+		c = framework.Client
```
Comment (wojtek-t) [inline test/e2e/density.go]: remove empty line after it

**Comment by wojtek-t** [comment]
LGTM - just one minor nit.

**Comment by wojtek-t** [comment]
@k8s-bot test this please github issue: #IGNORE (test didn't started)

**Comment by wojtek-t** [comment]
@gmarek - there are some compile errors - can you please fix them?

**Comment by gmarek** [comment]
It was caused by introduction of new e2e tests. Fixed now.
@wojtek-t PTAL

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**